### ISSUE TITLE
Unable to Navigate through Contextual Menu items in Browse Mode of NVDA

### ISSUE THREAD

**Issue by otaha2** [microsoft/fluentui#19741]
### Environment Information

- **Package version(s)**: NVDA
- **Browser and OS versions**: Edge Version 94.0.992.14 (Official build) beta (64-bit)

### Please provide a reproduction of the issue in a codepen:

Do not have a repro in a codepen at the moment

### Describe the issue:

Unable access/select the contextual menu items of command bar in browse mode of NVDA. Instead, when browse mode is enabled, NVDA focuses out of the contextual menu and to other background items on the screen. 

#### Actual behavior:

Users are unable access/select the contextual menu items of command bar in browse mode of NVDA
In Application mode (Normal mode) user is able to navigate over the listed items but after enabling browse mode, NVDA focus move to other background controls.

#### Expected behavior:

Users should be able to access/select the list items of contextual menu item using down arrow key and when user presses Enter , then the list item should get selected in browse mode of NVDA.

**Comment by gouttierre** [comment]
@otaha2 - Thanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the [ContextualMenu](https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu) examples (click “Export to CodePen”) or https://aka.ms/fluentpen.
